src/webui/webapplication.cpp

```diff
@@ -645,7 +645,7 @@ void WebApplication::sessionStart()
 
     // remove outdated sessions
     const qint64 now = QDateTime::currentMSecsSinceEpoch() / 1000;
-    foreach (const auto session, m_sessions) {
+    for (const auto session : qAsConst(m_sessions)) {
```

Comment:
I looked at all of them again and i only found this one. I'll make a copy of `m_sessions`.